codersdk/provisionerdaemons.go

```diff
@@ -114,6 +114,33 @@ const (
 	ProvisionerJobUnknown   ProvisionerJobStatus = "unknown"
 )
 
+func ProvisionerJobStatusEnums() []ProvisionerJobStatus {
+	return []ProvisionerJobStatus{
+		ProvisionerJobPending,
+		ProvisionerJobRunning,
+		ProvisionerJobSucceeded,
+		ProvisionerJobCanceling,
+		ProvisionerJobCanceled,
+		ProvisionerJobFailed,
+		ProvisionerJobUnknown,
+	}
+}
```

Comment:
Unused function?

Reply:
It's used in #16030, but ended up in this PR since it introduced the type.